specifyweb/frontend/js_src/lib/components/AttachmentsBulkImport/PerformAttachmentTask.ts

```diff
@@ -119,7 +118,7 @@ export function PerformAttachmentTask({
                 workRef.current.retrySpec[currentUploadingIndex] ??= 0;
                 const nextTry =
                   workRef.current.retrySpec[currentUploadingIndex];
-                workRef.current.retrySpec[currentUploadingIndex] += 1;
+                workRef.current.retrySpec[currentUploadingIndex] += SECOND;
```

Comment:
revert back to += 1

I used time units wherever needed, so it is safe to assume that no place (in any of batch upload) hard codes time!